### PR TITLE
v5.1.1 Runtime release

### PR DESCRIPTION
**Pull request by Gauthamastro**
## Describe your changes
Releases the final runtime and client for v5.1.1

### CODE REVIEW

File: pallets/thea-message-handler/src/benchmarking.rs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	verify {
+		assert_eq!(b, <IncomingNonce<T>>::get());
+	}
+}
+
+//#[cfg(test)]
+//use frame_benchmarking::impl_benchmark_test_suite;
+//
+//#[cfg(test)]
+//impl_benchmark_test_suite!(TheaMH, crate::mock::new_test_ext(), crate::mock::Test);
```
Comment (serhii-temchenko) [inline pallets/thea-message-handler/src/benchmarking.rs]: Should this be uncommented?

File: pallets/thea-message-handler/src/lib.rs [hunk 2]
```diff
@@ -27,18 +27,27 @@
...
 use thea_primitives::{types::Message, Network, ValidatorSet};
 
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+pub trait WeightInfo {
+	fn insert_authorities(_b: u32) -> Weight;
+	fn incoming_message() -> Weight;
+	fn update_incoming_nonce(_b: u32) -> Weight;
+}
```
Comment (serhii-temchenko) [inline pallets/thea-message-handler/src/lib.rs:48]: Adding required empty line.
```suggestion
}

```

File: pallets/thea-message-handler/src/lib.rs [hunk 3]
```diff
@@ -232,6 +244,21 @@ pub mod pallet {
 			<IncomingNonce<T>>::put(nonce);
 			Ok(())
 		}
+
+		/// A governance endpoint to update outgoing nonces
+		#[pallet::call_index(3)]
+		#[pallet::weight(Weight::default())]
```
Comment (serhii-temchenko) [inline pallets/thea-message-handler/src/lib.rs]: Should be added  benchmarks and weights?

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -1352,6 +1353,69 @@ impl thea_executor::Config for Runtime {
...
+		Recovery: pallet_recovery::{Pallet, Call, Storage, Event<T>} = 23,
+		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 24,
+		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 25,
+		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 26,
+		Bounties: pallet_bounties::{Pallet, Call, Storage, Event<T>} = 27,
+		OrmlVesting: orml_vesting::{Pallet, Storage, Call, Event<T>, Config<T>} = 28,
+		PDEXMigration: pdex_migration::pallet::{Pallet, Storage, Call, Event<T>, Config<T>} = 29,
+		Democracy: pallet_democracy::{Pallet, Call, Storage, Event<T>, Config<T>} = 30,
+		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 31,
+		//RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 32,
```
Comment (serhii-temchenko) [inline runtime/src/lib.rs:1405]: Should be removed?
```suggestion
```

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

File: pallets/thea-message-handler/src/lib.rs (hunk 3 above)
Comment (Gauthamastro) [reply pallets/thea-message-handler/src/lib.rs]: @zktony missed the benchmark for that extrinsic but it is technically same as update incoming nonce, so use it's weight for now.

File: runtime/src/lib.rs (hunk 4 above)
Comment (Gauthamastro) [reply runtime/src/lib.rs:1405]: No, until we finalize this change is okay, let it be there,

**Pull request merged**